src/plugins/analysis/ikconfig/test/test_ikconfig.py

```diff
@@ -0,0 +1,164 @@
+import glob
+import os
+
+from common_helper_files import get_dir_of_file
+
+from objects.file import FileObject
+from test.unit.analysis.analysis_plugin_test_class import AnalysisPluginTest
+from ..code.ikconfig import AnalysisPlugin
+from ..internal.decomp import GZDecompressor
+
+
+TEST_DATA_DIR = os.path.join(get_dir_of_file(__file__), 'data')
+
+
+class ExtractIKConfigTest(AnalysisPluginTest):
+    PLUGIN_NAME = 'ikconfig'
+
+    def setUp(self):
+        super().setUp()
+        config = self.init_basic_config()
+        self.analysis_plugin = AnalysisPlugin(self, config=config)
+
+    def test_object_mime_is_plaintext_true(self):
+        test_file = FileObject(file_path=os.path.join(TEST_DATA_DIR, 'configs/CONFIG'))
+        test_file.processed_analysis['file_type'] = dict(mime='text/plain')
+
+        assert(AnalysisPlugin.object_mime_is_plaintext(test_file))
```

Comment:
Please don't test static methods in a subclass of `AnalysisPluginTest` because the setup and teardown are not necessary and take a considerable amount of time

Also, the parentheses are not necessary:
```suggestion
        assert AnalysisPlugin.object_mime_is_plaintext(test_file)
```